### DISCUSSION TITLE
Body Armor (BAUHAUS) [C]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#26] [Card Balance]
Body Armor (BAUHAUS) [C]
EQUIPMENT, ARMOR
COMBAT enemies may not use a BLADE or FIREARM when combating this Warrior.

I suggest a litte bonus to [A] because  bauhaus warriors have a good armor.
➡️ +1 to [A]

**Comment by Sol-au-fein** [comment]
I would like to wait whether or not Venusian Marshall's Armor going to be lowered.

**Reply by Kite80** [reply]
We could set a general parameter limit for A for every warrior, like if you're made of flesh and have a cuirass your A can't be higher than 6 or 7, otherwise you're a machine or a DL monster... at that point we can nerf warriors with eccessive A (Cardinal Dominic could be an exception, but after all [spoiler alert!] he's not human)

**Comment by Sol-au-fein** [comment]
In paper doomtrooper it has +3A and immunity to weapons. 
I think +1A and immunity to weapons in digital version would be sufficient, considering all armors are bit nerfed in digital. (Carcass suit is +5A, armor of true assassin is +4A, composite armor +4A)
